src/Migration/Migration1605714035FinSearchConfig.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Migration;
+
+use Doctrine\DBAL\Connection;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Migration\MigrationStep;
+use Shopware\Core\Framework\Uuid\Uuid;
+
+class Migration1605714035FinSearchConfig extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1605714035;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $sql = 'CREATE TABLE IF NOT EXISTS `finsearch_config` (
```

Comment:
Use [heredoc syntax](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) when writing a string in multiple lines.